### DISCUSSION TITLE
辽宁地区 官网貌似被污染了....直连不上哪 得改host  改了之后官网能上了但客户端登不上去....

### DISCUSSION THREAD

**Discussion by 373578807** [natfrp/wiki#287] [使用问题] [answered]
辽宁地区 官网貌似被dns污染了....直连不上哪 得改host或者爬梯子  改了之后官网能上了但客户端登不上去.... 而且也容易出问题 有什么好的解决办法吗？

**Comment by 2one9** [comment]
好像确实是这样，我也是辽宁的。昨天晚上还可以使用，今天就不可以了，我宽带和流量都试过了，都没法用，但是我朋友还可以用。

**Comment by fakeboboliu** [comment, marked as answer]
感谢报告，我们注意到在很多地方又出现了对我们服务的 DNS 污染情况，并且正在讨论解决方案。

对于目前，您可以在 hosts 中指定两个域名：`www.natfrp.com` 用于我们官网的方案和 `api.natfrp.com` 用于客户端。

例如：
```
111.31.58.195 www.natfrp.com
111.31.58.195 api.natfrp.com
```

**Reply by 2one9** [reply]
感谢回复，我是纯小白，请问修改host时前面需要加的ip如何获取呢

**Reply by 2one9** [reply]
感谢回复，我现在成功连接到了官网与客户端。但是上面提供的网址告诉我的ip地址并不能用，不过我让我的朋友帮忙ping了一下成功获得了ip地址，现在我已经可以正常使用樱花穿透了

**Reply by 2one9** [reply]
感谢回复，我现在成功连接到了官网与客户端。不过官网的样子好像并没有复原，如下图；另外上面提供的网址告诉我的ip地址并不能用，不过我让我的朋友帮忙ping了一下成功获得了ip地址，现在我已经可以正常使用樱花穿透了
![9IN6Y 6~94Z QOY647%76CI](https://user-images.githubusercontent.com/62428860/221364140-817be40d-7e8f-4787-8b6c-d9b2cc6d826b.png)

**Reply by 373578807** [reply]
你好 我也是个小白 方便分享一下可用的ip和需要的域名吗...我现在反而是能登录官网 但因为cf完全不响应 导致客户端没法登录

**Reply by fakeboboliu** [reply]
> 你好 我也是个小白 方便分享一下可用的ip和需要的域名吗...我现在反而是能登录官网 但因为cf完全不响应 导致客户端没法登录

您好，我已经在主回答中更新一个 hosts 配置示例，您可以进行参考

**Reply by 373578807** [reply]
好的 十分感谢！

[a comment by an automated account is not shown]
